UI/UnoSQLiteOneDriveInvoiceSample/Demo/Demo/Demo.Shared/Pages/Clients.xaml.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Demo.Database.Entities;
+using Demo.ViewModels;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace Demo.Pages
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class Clients : Page
```

Comment:
How is the DataContext being set for this page?

Reply:
It's set in a child of TabView of the MainPage.xaml